plugin/path_config_test.go

```diff
@@ -31,9 +33,17 @@ func TestBackend_PathConfigRead(t *testing.T) {
 		if err != nil {
 			t.Fatal(err)
 		}
-
-		if resp != nil {
-			t.Errorf("expected response data to be empty, got %v", resp.Data)
+		if resp == nil {
```

Comment:
Can an expected response object be created and then use `reflect.DeepEqual` instead of these checks?

Reply:
Will this work instead of multiple separate assertions?
```go
expectedResp := &logical.Response{
  Data: map[string]interface{}{
    "iam_metadata": []string{
      ...
    },
    "gce_metadata": []string{
      ...
    },
  },
}

if !reflect.DeepEqual(resp, expectedResp) {
  t.Fatalf("Actual: %#v\nExpected: %#v", resp, expectedResp)
}
```

Reply:
Sure, pushed in https://github.com/hashicorp/vault-plugin-auth-gcp/pull/92/commits/2495f0ee52f7997018c9e11e1cf53bacd2a328aa